etc/skydive.yml.default

```diff
@@ -342,10 +341,20 @@ flow:
   # Protocol to use to send flows to the analyzer: websocket or udp
   # protocol: udp
 
+  # Set the application field according to the following port mapping
+  application_ports:
```

Comment:
I think it is naive and that the goal, adding more mapping stuff will be a bit tricky in the conf and won't be exhaustive neither. The goal is to provide something similar to /etc/services. It works per agent. For the flow enhancer, It will be more difficult as we don't have a way to know if the "resolution" would have been done or not as we override the Application field.

For something more complex I think it would be better to provide a way to script the classifier as we discussed during the hackathon.

Reply:
@lindhor hi, is the current approach fit your needs ?

Reply:
Sorry for the delayed answer, I’m out travelling. Unfortunately I’m currently not able to test it right now but I looked at the code changes and as far as I can tell it would be enough for my purpose. Of course it would be nice to have a GUI where unidentified traffic can be added and a possibility to differentiate on higher layers than tcp/udp but that could come later on. We would benefit a lot from this change. If not too much effort you might consider adding support for icmp with message types while at it. Also I can’t read out from the code if IPv6 will be supported (not an issue for me but anyway).
// Roger